src/porepy/fracs/fractures_2d.py

```diff
@@ -531,6 +533,31 @@ def _insert_auxiliary_points(
         self._decomposition["edges"] = lines
         self._decomposition["mesh_size"] = mesh_size
 
+    @pp.time_logger(sections=module_sections)
+    def _set_dfn_mesh_size(
```

Comment:
This is needed because the _insert_auxillary_points also defines the mesh size. This is a bit of a hack, and I think a better approach would be to refactor the _insert_auxillary_points into a function for inserting the points and one for determining the mesh size. However, the mesh size determination is so interwoven with the insertion of the points in pp.fracs.tools.determine_mesh_size(..) that I was not brave enough to tackle this challenge.

I will however, add some comments about this

Reply:
Also do you have a better name for it? Could we maybe call them:
_set_mesh_size_with_auxiliary_points()
and
_set_mesh_size_without_auxilary_points() ?

Reply:
I decided to change _set_dfn_mesh_size -> set_mesh_size_without_auxiliary_points().  I will wait for @keileg 's feedback before renaming  _insert_auxiliary_points

Reply:
Let's keep it like it is now. The idea is that the meshing should do two things:
1) Split all intersections, so that gmsh does not complain
2) Attempt to set reasonable mesh sizes with a minimal effort from the user. 
If one wants something more, the best option is to interact with gmsh directly, and I don't think it is a good idea to expose too many options on the PorePy side - that will soon be complex to maintain.

The dfn case you are working on deserves some special treatment, and if we ever get better way to interact with the gmsh python api, we may expand the new option to non-dfn cases as well, but until we see more of a need, I'm happy with the current structure.